pypeit/par/pypeitpar.py

```diff
@@ -1726,6 +1726,10 @@ def __init__(self, obj_toler=None, assign_obj=None, nsig_thrshd=None,
         dtypes['assign_obj'] = bool
         descr['assign_obj'] = 'If SlitMask object was generated, assign RA,DEC,name to detected objects'
 
+        defaults['skip_serendip'] = False
```

Comment:
Let's discuss if this is needed. I feel strongly it makes no sense given how local is performed.